.github/workflows/e2e.yml

```diff
@@ -36,6 +36,8 @@ jobs:
         working-directory: ./packages/create-llama
       - name: Run Playwright tests
         run: pnpm exec playwright test
+        env: 
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
```

Comment:
@yisding please check if `OPENAI_API_KEY` is set correctly, as e2e tests are not running

Reply:
It says it was set 4 days ago, but I tried adding it again (with a different key)